index.html

```diff
@@ -67,11 +67,7 @@ <h1>Welcome to Jason Leveille's CSD120 Website!</h1>
                         <th scope="col">Hrs/Week</th>
                     </tr>
                 </thead>
-<<<<<<< HEAD
-                <tbody>                                      
-=======
                 <tbody>
->>>>>>> 3096361c54ce560ec00f5bd9cb84ba952535c113
                     <tr>
```

Comment:
Make sure you remove the conflict markers too when you are dealing with merge conflicts!